pkg/limayaml/limayaml.go

```diff
@@ -107,14 +107,19 @@ type User struct {
 }
 
 type VMOpts struct {
-	QEMU QEMUOpts `yaml:"qemu,omitempty" json:"qemu,omitempty"`
+	QEMU   QEMUOpts   `yaml:"qemu,omitempty" json:"qemu,omitempty"`
+	Docker DockerOpts `yaml:"docker,omitempty" json:"docker,omitempty"`
 }
```

Comment:
This contradicts to the current VM driver design.
Driver details must not be put to the yaml definition

Reply:
- https://github.com/lima-vm/lima/issues/3506